Documentation/CrossPlatform/LeapMotionMRTK.md

```diff
@@ -53,6 +58,36 @@ The Leap Motion Data Provider enables articulated hand tracking for VR and could
     - Navigate to **File > Build Settings**
     - Only Standalone builds are supported if using the Leap Motion Data Provider.
```

Comment:
Step by step documentation for unity mrtk users #7685 will be linked here after it is complete.